File: vSongApp/src/main/java/com/jackson_siro/visongbook/models/Callback/CallbackPostsSearch.java
package com.jackson_siro.visongbook.models.Callback;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

import com.jackson_siro.visongbook.models.PostsSearch;

public class CallbackPostsSearch implements Serializable {
    public List<PostsSearch> data = new ArrayList<PostsSearch>();
}

File: vSongApp/src/main/java/com/jackson_siro/visongbook/models/callback/CallbackPostsSearch.kt
package com.jackson_siro.visongbook.models.callback

import java.util.ArrayList
import java.io.Serializable
import com.jackson_siro.visongbook.models.PostsSearch

internal class CallbackPostsSearch : Serializable {
    var data: List<PostsSearch> = ArrayList()
}
